src/Docker.PowerShell/Cmdlets/NetworkArgumentCompleter.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Management.Automation.Language;
+using System.Linq;
+using Docker.DotNet.Models;
+using System;
+
+public class NetworkArgumentCompleter : IArgumentCompleter
+{
+    public IEnumerable<CompletionResult> CompleteArgument(string commandName,
+                                                          string parameterName,
+                                                          string wordToComplete,
+                                                          CommandAst commandAst,
+                                                          IDictionary fakeBoundParameters)
+    {
+        var client = DockerFactory.CreateClient(fakeBoundParameters);
+        IList<NetworkListResponse> result;
```

Comment:
This used to be in an if-statement, but when I removed that I forgot to simplify.